cumulus/parachains/runtimes/assets/asset-hub-next-westend/README.md

```diff
@@ -13,15 +13,37 @@ Build/install binaries from above.
 
 Then:
 
-```
+```bash
 cargo build --release -p asset-hub-next-westend-runtime -p staging-chain-spec-builder
 ../../../../../target/release/chain-spec-builder create --runtime ../../../../../target/release/wbuild/asset-hub-next-westend-runtime/asset_hub_next_westend_runtime.compact.compressed.wasm --relay-chain rococo-local --para-id 1100 named-preset genesis
```

Comment:
I'd hugely recommend people to keep using polkadot-sdk as their working directory for these tests, which is why I originally had these as just ./target
```suggestion
./target/release/chain-spec-builder create --runtime ./target/release/wbuild/asset-hub-next-westend-runtime/asset_hub_next_westend_runtime.compact.compressed.wasm --relay-chain westend-local --para-id 1100 named-preset development
```

Also why was it changed to rococo (not in your branch)? 🧐

Reply:
yea just seems like a clobber from a previous merge